fdbserver/BackupProgress.actor.cpp

```diff
@@ -156,12 +156,14 @@ ACTOR Future<Void> getBackupProgress(Database cx, UID dbgid, Reference<BackupPro
 				const UID workerID = decodeBackupProgressKey(it.key);
 				const WorkerBackupStatus status = decodeBackupProgressValue(it.value);
 				bStatus->addBackupStatus(status);
-				TraceEvent("GotBackupProgress", dbgid)
-				    .detail("BackupWorker", workerID)
-				    .detail("Epoch", status.epoch)
-				    .detail("Version", status.version)
-				    .detail("Tag", status.tag.toString())
-				    .detail("TotalTags", status.totalTags);
+				if (logging) {
+					TraceEvent("GotBackupProgress", dbgid)
```

Comment:
Maybe add a suppressFor() so that it won't be spammy when we enable logging

Reply:
This event is important to Master and should be printed out during master recovery. In contrast, BackupWorkers also call this function, but don't really need the events. Since there are (potentially) many BackupWorkers and each one is calling this periodically, this event become quite verbose. So disabling for BackupWorkers should be good enough.